### PR TITLE
Vastly improve authenticity

### PR DESCRIPTION
**Pull request by kyunal**
The given commit should vastly improve your blogs authenticity by adding valid reasoning as to why you should be trusted over others. I'd say this is a significant step in order to make things more believable. In fact, this small addition makes it so you outperform every other blog of this kind by a vast margin in this specific regard. It'll also make up for a much more confident appearance of your own, while being incredibly unique at it. For all these reasons, I don't think there's any more room left to negotiate.

To summarize: Add this. I appreciate your consideration.

### CODE REVIEW

**Comment by madaidans-insecurities** [comment]
image
